policy/modules/xen/ndvm.te

```diff
@@ -24,6 +24,7 @@ policy_module(ndvm, 0.2)
 #
 
 type ndvm_t;
+typealias ndvm_t alias { usbvm_t };
```

Comment:
leaving the usbvm stuff in here is fine, nothing is making use of it until we merge the main usbvm stuff.